linked-lists/c-or-cpp/josephus-problem.c

```diff
@@ -0,0 +1,63 @@
+/*Circular linked list implementation in C Programming
```

Comment:
It would be better if you could add a bit of description of the problem on the top.